pages/getting-started/index.js

```diff
@@ -57,7 +57,7 @@ function CodePreview() {
 
 function Page({url}) {
   return (
-    <Layout pathname={url ? url.pathname : ''}>
+    <Layout pathname={url ? url.pathname : ''} topNav={true}>
```

Comment:
Only applies to `getting-started` and `index`. All the others are not on the top.